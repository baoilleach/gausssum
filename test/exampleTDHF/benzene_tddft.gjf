%chk=benzene.chk
# HF/3-21G guess=read geom=allcheck td