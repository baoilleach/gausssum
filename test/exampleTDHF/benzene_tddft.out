 Entering Link 1 = C:\G03W\l1.exe PID=      1284.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2004,2007, Gaussian, Inc.
                  All Rights Reserved.
  
 This is the Gaussian(R) 03 program.  It is based on the
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 03, Revision E.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, J. A. Montgomery, Jr., T. Vreven, 
 K. N. Kudin, J. C. Burant, J. M. Millam, S. S. Iyengar, J. Tomasi, 
 V. Barone, B. Mennucci, M. Cossi, G. Scalmani, N. Rega, 
 G. A. Petersson, H. Nakatsuji, M. Hada, M. Ehara, K. Toyota, 
 R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, 
 H. Nakai, M. Klene, X. Li, J. E. Knox, H. P. Hratchian, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 P. Y. Ayala, K. Morokuma, G. A. Voth, P. Salvador, J. J. Dannenberg, 
 V. G. Zakrzewski, S. Dapprich, A. D. Daniels, M. C. Strain, 
 O. Farkas, D. K. Malick, A. D. Rabuck, K. Raghavachari, 
 J. B. Foresman, J. V. Ortiz, Q. Cui, A. G. Baboul, S. Clifford, 
 J. Cioslowski, B. B. Stefanov, G. Liu, A. Liashenko, P. Piskorz, 
 I. Komaromi, R. L. Martin, D. J. Fox, T. Keith, M. A. Al-Laham, 
 C. Y. Peng, A. Nanayakkara, M. Challacombe, P. M. W. Gill, 
 B. Johnson, W. Chen, M. W. Wong, C. Gonzalez, and J. A. Pople, 
 Gaussian, Inc., Wallingford CT, 2004.
 
 ******************************************
 Gaussian 03:  IA32W-G03RevE.01 11-Sep-2007
                20-May-2009 
 ******************************************
 %chk=benzene.chk
 --------------------------------------
 # HF/3-21G guess=read geom=allcheck td
 --------------------------------------
 1/29=7,38=1/1;
 2/40=1/2;
 3/5=5,11=9,16=1,25=1,30=1/1,2,8,3;
 4/5=1/1;
 5/5=2,38=6/2;
 8/6=1,10=1/1;
 9/42=1,70=1/14;
 6/7=2,8=2,9=2,10=2/1;
 99/5=1,9=1/99;
 --------
 _Benzene
 --------
 Redundant internal coordinates taken from checkpoint file:
 benzene.chk
 Charge =  0 Multiplicity = 1
 H,0,0.,0.,0.0290732178
 C,0,0.,0.,1.101090291
 C,0,1.1991937848,0.,1.7934451455
 H,0,2.1275878035,0.,1.2574366089
 C,0,1.1991937848,0.,3.1781548545
 H,0,2.1275878035,0.,3.7141633911
 C,0,0.,0.,3.870509709
 H,0,0.,0.,4.9425267822
 C,0,-1.1991937848,0.,3.1781548545
 H,0,-2.1275878035,0.,3.7141633911
 C,0,-1.1991937848,0.,1.7934451455
 H,0,-2.1275878035,0.,1.2574366089
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic     Atomic              Coordinates (Angstroms)
 Number     Number      Type              X           Y           Z
 ---------------------------------------------------------------------
    1          1             0        0.000000    0.000000    0.029073
    2          6             0        0.000000    0.000000    1.101090
    3          6             0        1.199194    0.000000    1.793445
    4          1             0        2.127588    0.000000    1.257437
    5          6             0        1.199194    0.000000    3.178155
    6          1             0        2.127588    0.000000    3.714163
    7          6             0        0.000000    0.000000    3.870510
    8          1             0        0.000000    0.000000    4.942527
    9          6             0       -1.199194    0.000000    3.178155
   10          1             0       -2.127588    0.000000    3.714163
   11          6             0       -1.199194    0.000000    1.793445
   12          1             0       -2.127588    0.000000    1.257437
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  H    0.000000
     2  C    1.072017   0.000000
     3  C    2.133325   1.384710   0.000000
     4  H    2.456727   2.133325   1.072017   0.000000
     5  C    3.369686   2.398388   1.384710   2.133325   0.000000
     6  H    4.255176   3.369686   2.133325   2.456727   1.072017
     7  C    3.841436   2.769419   2.398388   3.369686   1.384710
     8  H    4.913454   3.841436   3.369686   4.255176   2.133325
     9  C    3.369686   2.398388   2.769419   3.841436   2.398388
    10  H    4.255176   3.369686   3.841436   4.913454   3.369686
    11  C    2.133325   1.384710   2.398388   3.369686   2.769419
    12  H    2.456727   2.133325   3.369686   4.255176   3.841436
                    6          7          8          9         10
     6  H    0.000000
     7  C    2.133325   0.000000
     8  H    2.456727   1.072017   0.000000
     9  C    3.369686   1.384710   2.133325   0.000000
    10  H    4.255176   2.133325   2.456727   1.072017   0.000000
    11  C    3.841436   2.398388   3.369686   1.384710   2.133325
    12  H    4.913454   3.369686   4.255176   2.133325   2.456727
                   11         12
    11  C    0.000000
    12  H    1.072017   0.000000
 Stoichiometry    C6H6
 Framework group  D6H[3C2'(HC.CH)]
 Deg. of freedom     2
 Full point group                 D6H
 Largest Abelian subgroup         D2H     NOp   8
 Largest concise Abelian subgroup D2      NOp   4
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic     Atomic              Coordinates (Angstroms)
 Number     Number      Type              X           Y           Z
 ---------------------------------------------------------------------
    1          1             0        0.000000    2.456727    0.000000
    2          6             0        0.000000    1.384710    0.000000
    3          6             0        1.199194    0.692355    0.000000
    4          1             0        2.127588    1.228363    0.000000
    5          6             0        1.199194   -0.692355    0.000000
    6          1             0        2.127588   -1.228363    0.000000
    7          6             0        0.000000   -1.384710    0.000000
    8          1             0        0.000000   -2.456727    0.000000
    9          6             0       -1.199194   -0.692355    0.000000
   10          1             0       -2.127588   -1.228363    0.000000
   11          6             0       -1.199194    0.692355    0.000000
   12          1             0       -2.127588    1.228363    0.000000
 ---------------------------------------------------------------------
 Rotational constants (GHZ):      5.7906265      5.7906265      2.8953132
 Standard basis: 3-21G (6D, 7F)
 There are    16 symmetry adapted basis functions of AG  symmetry.
 There are    11 symmetry adapted basis functions of B1G symmetry.
 There are     2 symmetry adapted basis functions of B2G symmetry.
 There are     4 symmetry adapted basis functions of B3G symmetry.
 There are     2 symmetry adapted basis functions of AU  symmetry.
 There are     4 symmetry adapted basis functions of B1U symmetry.
 There are    16 symmetry adapted basis functions of B2U symmetry.
 There are    11 symmetry adapted basis functions of B3U symmetry.
 Integral buffers will be    262144 words long.
 Raffenetti 1 integral format.
 Two-electron integral symmetry is turned on.
    66 basis functions,   108 primitive gaussians,    66 cartesian basis functions
    21 alpha electrons       21 beta electrons
       nuclear repulsion energy       205.0986500017 Hartrees.
 NAtoms=   12 NActive=   12 NUniq=    2 SFac= 3.00D+00 NAtFMM=   80 NAOKFM=F Big=F
 One-electron integrals computed using PRISM.
 NBasis=    66 RedAO= T  NBF=    16    11     2     4     2     4    16    11
 NBsUse=    66 1.00D-06 NBFU=    16    11     2     4     2     4    16    11
 Initial guess read from the checkpoint file:
 benzene.chk
 Initial guess orbital symmetries:
       Occupied  (B1U) (E2G) (E2G) (E1U) (E1U) (A1G) (A1G) (E1U)
                 (E1U) (E2G) (E2G) (A1G) (B1U) (B2U) (E1U) (E1U)
                 (A2U) (E2G) (E2G) (E1G) (E1G)
       Virtual   (E2U) (E2U) (A1G) (E1U) (E1U) (E2G) (E2G) (B2G)
                 (B1U) (E2G) (E2G) (E1U) (E1U) (A2G) (B1U) (B2U)
                 (A1G) (A2U) (E2G) (E2G) (E1G) (E1G) (E1U) (E1U)
                 (E2U) (E2U) (A1G) (B2G) (B1U) (E2G) (E2G) (E1U)
                 (E1U) (A2G) (E1U) (E1U) (A1G) (B1U) (E2G) (E2G)
                 (E1U) (E1U) (E2G) (E2G) (B1U)
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 Keep R1 integrals in memory in canonical form, NReq=     3425952.
 SCF Done:  E(RHF) =  -229.419445317     A.U. after    1 cycles
             Convg  =    0.5987D-09             -V/T =  2.0017
             S**2   =   0.0000
 ExpMin= 1.83D-01 ExpMax= 1.72D+02 ExpMxC= 1.72D+02 IAcc=1 IRadAn=         1 AccDes= 1.00D-06
 HarFok:  IExCor= 205 AccDes= 1.00D-06 IRadAn=         1 IDoV=1
 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     7    66
 NBasis=    66 NAE=    21 NBE=    21 NFC=     6 NFV=     0
 NROrb=     60 NOA=    15 NOB=    15 NVA=    45 NVB=    45
 Would need an additional     1844642 words for in-memory AO integral storage.
 Orbital symmetries:
       Occupied  (B1U) (E2G) (E2G) (E1U) (E1U) (A1G) (A1G) (E1U)
                 (E1U) (E2G) (E2G) (A1G) (B1U) (B2U) (E1U) (E1U)
                 (A2U) (E2G) (E2G) (E1G) (E1G)
       Virtual   (E2U) (E2U) (A1G) (E1U) (E1U) (E2G) (E2G) (B2G)
                 (B1U) (E2G) (E2G) (E1U) (E1U) (A2G) (B1U) (B2U)
                 (A1G) (A2U) (E2G) (E2G) (E1G) (E1G) (E1U) (E1U)
                 (E2U) (E2U) (A1G) (B2G) (B1U) (E2G) (E2G) (E1U)
                 (E1U) (A2G) (E1U) (E1U) (A1G) (B1U) (E2G) (E2G)
                 (E1U) (E1U) (E2G) (E2G) (B1U)
    24 initial guesses have been made.
 Convergence on wavefunction:    0.001000000000000
 Iteration     1 Dimension    24 NMult    24
 CISAX will form  24 AO SS matrices at one time.
 New state      1 was old state      3
 New state      2 was old state      1
 New state      3 was old state      1
 Iteration     2 Dimension    30 NMult    30
 Iteration     3 Dimension    36 NMult    36
 Iteration     4 Dimension    40 NMult    40
 Iteration     5 Dimension    44 NMult    44
 ***********************************************************************
 Excited states from <AA,BB:AA,BB> singles matrix:
 ***********************************************************************
 
 Ground to excited state Transition electric dipole moments (Au):
       state          X           Y           Z          Osc.
         1         0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000      0.0000
         3         0.1658     -1.8751      0.0000      0.7186
 Ground to excited state transition velocity dipole Moments (Au):
       state          X           Y           Z          Osc.
         1         0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000      0.0000
         3        -0.0434      0.4910      0.0000      0.5325
 Ground to excited state transition magnetic dipole Moments (Au):
       state          X           Y           Z
         1         0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000
         3         0.0000      0.0000      0.0000
 <0|del|b> * <b|rxdel|0>  (Au), Rotatory Strengths (R) in
 cgs (10**-40 erg-esu-cm/Gauss)
       state          X           Y           Z     R(velocity)
         1         0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000      0.0000
         3         0.0000      0.0000      0.0000      0.0000
 <0|r|b> * <b|rxdel|0>  (Au), Rotatory Strengths (R) in
 cgs (10**-40 erg-esu-cm/Gauss)
       state          X           Y           Z     R(length)
         1         0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000      0.0000
         3         0.0000      0.0000      0.0000      0.0000
  <0|del|b> * <b|r|0>  (Au)
       state          X           Y           Z       Osc.(frdel)
         1         0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000      0.0000
         3        -0.0072     -0.9207      0.0000      0.6186
 Ground to excited state transition densities written to RWF  633

 Excitation energies and oscillator strengths:
 
 Excited state symmetry could not be determined.
 Excited State   1:   Singlet-?Sym    6.4703 eV  191.62 nm  f=0.0000
      20 -> 22         0.55704
      21 -> 23         0.55704
 This state for optimization and/or second-order correction.
 Copying the excited state density for this state as the 1-particle RhoCI density.
 
 Excited state symmetry could not be determined.
 Excited State   2:   Singlet-?Sym    6.5702 eV  188.71 nm  f=0.0000
      20 -> 23        -0.52281
      21 -> 22         0.52281
 
 Excited state symmetry could not be determined.
 Excited State   3:   Singlet-?Sym    8.2776 eV  149.78 nm  f=0.7186
      20 -> 23         0.42888
      21 -> 22         0.42888

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (B1U) (E2G) (E2G) (E1U) (E1U) (A1G) (A1G) (E1U)
                 (E1U) (E2G) (E2G) (A1G) (B1U) (B2U) (E1U) (E1U)
                 (A2U) (E2G) (E2G) (E1G) (E1G)
       Virtual   (E2U) (E2U) (A1G) (E1U) (E1U) (E2G) (E2G) (B2G)
                 (B1U) (E2G) (E2G) (E1U) (E1U) (A2G) (B1U) (B2U)
                 (A1G) (A2U) (E2G) (E2G) (E1G) (E1G) (E1U) (E1U)
                 (E2U) (E2U) (A1G) (B2G) (B1U) (E2G) (E2G) (E1U)
                 (E1U) (A2G) (E1U) (E1U) (A1G) (B1U) (E2G) (E2G)
                 (E1U) (E1U) (E2G) (E2G) (B1U)
 The electronic state is 1-A1G.
 Alpha  occ. eigenvalues --  -11.17550 -11.17527 -11.17527 -11.17506 -11.17506
 Alpha  occ. eigenvalues --  -11.17449  -1.15729  -1.01648  -1.01648  -0.82563
 Alpha  occ. eigenvalues --   -0.82563  -0.71991  -0.63955  -0.62488  -0.59281
 Alpha  occ. eigenvalues --   -0.59281  -0.50842  -0.49145  -0.49145  -0.33897
 Alpha  occ. eigenvalues --   -0.33897
 Alpha virt. eigenvalues --    0.15121   0.15121   0.25741   0.32454   0.32454
 Alpha virt. eigenvalues --    0.37203   0.37203   0.37542   0.37566   0.50850
 Alpha virt. eigenvalues --    0.50850   0.52423   0.52423   0.81331   0.83500
 Alpha virt. eigenvalues --    0.87045   0.90676   0.95557   1.00751   1.00751
 Alpha virt. eigenvalues --    1.05372   1.05372   1.08856   1.08856   1.10793
 Alpha virt. eigenvalues --    1.10793   1.13659   1.17875   1.29043   1.31644
 Alpha virt. eigenvalues --    1.31644   1.41517   1.41517   1.42398   1.55806
 Alpha virt. eigenvalues --    1.55806   1.59758   1.72308   1.73305   1.73305
 Alpha virt. eigenvalues --    1.81231   1.81231   2.17259   2.17259   2.77495
          Condensed to atoms (all electrons):
              1          2          3          4          5          6
     1  H    0.424097   0.399554  -0.032383  -0.001364   0.002353  -0.000036
     2  C    0.399554   5.177755   0.471510  -0.032383  -0.109810   0.002353
     3  C   -0.032383   0.471510   5.177755   0.399554   0.471510  -0.032383
     4  H   -0.001364  -0.032383   0.399554   0.424097  -0.032383  -0.001364
     5  C    0.002353  -0.109810   0.471510  -0.032383   5.177755   0.399554
     6  H   -0.000036   0.002353  -0.032383  -0.001364   0.399554   0.424097
     7  C   -0.000104  -0.001230  -0.109810   0.002353   0.471510  -0.032383
     8  H    0.000001  -0.000104   0.002353  -0.000036  -0.032383  -0.001364
     9  C    0.002353  -0.109810  -0.001230  -0.000104  -0.109810   0.002353
    10  H   -0.000036   0.002353  -0.000104   0.000001   0.002353  -0.000036
    11  C   -0.032383   0.471510  -0.109810   0.002353  -0.001230  -0.000104
    12  H   -0.001364  -0.032383   0.002353  -0.000036  -0.000104   0.000001
              7          8          9         10         11         12
     1  H   -0.000104   0.000001   0.002353  -0.000036  -0.032383  -0.001364
     2  C   -0.001230  -0.000104  -0.109810   0.002353   0.471510  -0.032383
     3  C   -0.109810   0.002353  -0.001230  -0.000104  -0.109810   0.002353
     4  H    0.002353  -0.000036  -0.000104   0.000001   0.002353  -0.000036
     5  C    0.471510  -0.032383  -0.109810   0.002353  -0.001230  -0.000104
     6  H   -0.032383  -0.001364   0.002353  -0.000036  -0.000104   0.000001
     7  C    5.177755   0.399554   0.471510  -0.032383  -0.109810   0.002353
     8  H    0.399554   0.424097  -0.032383  -0.001364   0.002353  -0.000036
     9  C    0.471510  -0.032383   5.177755   0.399554   0.471510  -0.032383
    10  H   -0.032383  -0.001364   0.399554   0.424097  -0.032383  -0.001364
    11  C   -0.109810   0.002353   0.471510  -0.032383   5.177755   0.399554
    12  H    0.002353  -0.000036  -0.032383  -0.001364   0.399554   0.424097
 Mulliken atomic charges:
              1
     1  H    0.239314
     2  C   -0.239314
     3  C   -0.239314
     4  H    0.239314
     5  C   -0.239314
     6  H    0.239314
     7  C   -0.239314
     8  H    0.239314
     9  C   -0.239314
    10  H    0.239314
    11  C   -0.239314
    12  H    0.239314
 Sum of Mulliken charges=   0.00000
 Atomic charges with hydrogens summed into heavy atoms:
              1
     1  H    0.000000
     2  C    0.000000
     3  C    0.000000
     4  H    0.000000
     5  C    0.000000
     6  H    0.000000
     7  C    0.000000
     8  H    0.000000
     9  C    0.000000
    10  H    0.000000
    11  C    0.000000
    12  H    0.000000
 Sum of Mulliken charges=   0.00000
 Electronic spatial extent (au):  <R**2>=   453.0868
 Charge=     0.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=     0.0000    Y=     0.0000    Z=     0.0000  Tot=     0.0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=   -31.4704   YY=   -31.4704   ZZ=   -40.9864
   XY=     0.0000   XZ=     0.0000   YZ=     0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=     3.1720   YY=     3.1720   ZZ=    -6.3440
   XY=     0.0000   XZ=     0.0000   YZ=     0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=     0.0000  YYY=     0.0000  ZZZ=     0.0000  XYY=     0.0000
  XXY=     0.0000  XXZ=     0.0000  XZZ=     0.0000  YZZ=     0.0000
  YYZ=     0.0000  XYZ=     0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=  -257.1204 YYYY=  -257.1204 ZZZZ=   -43.7275 XXXY=     0.0000
 XXXZ=     0.0000 YYYX=     0.0000 YYYZ=     0.0000 ZZZX=     0.0000
 ZZZY=     0.0000 XXYY=   -85.7068 XXZZ=   -61.7288 YYZZ=   -61.7288
 XXYZ=     0.0000 YYXZ=     0.0000 ZZXY=     0.0000
 N-N= 2.050986500017D+02 E-N=-9.435980861716D+02  KE= 2.290246098872D+02
 Symmetry AG   KE= 7.393320396291D+01
 Symmetry B1G  KE= 3.739413054505D+01
 Symmetry B2G  KE= 2.068581406077D+00
 Symmetry B3G  KE= 2.068581406077D+00
 Symmetry AU   KE=-2.725777292780D-16
 Symmetry B1U  KE= 1.748795694325D+00
 Symmetry B2U  KE= 7.160713560874D+01
 Symmetry B3U  KE= 4.020418126406D+01
 1|1|UNPC-UNK|SP|RTD-HF-FC|3-21G|C6H6|PCUSER|20-May-2009|0||# HF/3-21G 
 guess=read geom=allcheck td||_Benzene||0,1|H,0,0.,0.,0.0290732178|C,0,
 0.,0.,1.101090291|C,0,1.1991937848,0.,1.7934451455|H,0,2.1275878035,0.
 ,1.2574366089|C,0,1.1991937848,0.,3.1781548545|H,0,2.1275878035,0.,3.7
 141633911|C,0,0.,0.,3.870509709|H,0,0.,0.,4.9425267822|C,0,-1.19919378
 48,0.,3.1781548545|H,0,-2.1275878035,0.,3.7141633911|C,0,-1.1991937848
 ,0.,1.7934451455|H,0,-2.1275878035,0.,1.2574366089||Version=IA32W-G03R
 evE.01|State=1-A1G|HF=-229.4194453|RMSD=5.987e-010|Thermal=0.|PG=D06H 
 [3C2'(H1C1.C1H1)]||@


 LOGIC IS A SYSTEMATIC METHOD OF COMING TO THE
 WRONG CONCLUSION WITH CONFIDENCE.
 Job cpu time:  0 days  0 hours  0 minutes  6.0 seconds.
 File lengths (MBytes):  RWF=     22 Int=      0 D2E=      0 Chk=      9 Scr=      1
 Normal termination of Gaussian 03 at Wed May 20 12:29:24 2009.
